src/WebJobs.Script.WebHost/Helpers/DelegatingStream.cs

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Helpers
+{
+    public abstract class DelegatingStream : Stream
+    {
+        private Stream _innerStream;
+
+        protected DelegatingStream(Stream innerStream)
+        {
+            if (innerStream == null)
+            {
+                throw new ArgumentNullException("innerStream");
```

Comment:
nit: use `nameof` for parameter name and throw expression for the `null` validation.

Reply:
removed file